content/es/edge-computing.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Computación en el borde
```

Comment:
No estoy seguro de cuál sería la mejor traducción para Edge computing

* [Computación frontera](https://es.wikipedia.org/wiki/Computaci%C3%B3n_frontera)
* [Proceso perimetral](https://azure.microsoft.com/es-es/resources/cloud-computing-dictionary/what-is-edge-computing/)
* [Computación periférica](https://aws.amazon.com/es/what-is/edge-computing/)

cc @ramrodo @krol3 @92nqb @raelga

Reply:
Yo descartaria proceso perimetral. La pagina de Azure no parece tener una traducción muy cuidada.

Reply:
En mi opinón suena mejor la palabra **borde** ya que se refiere a colocar lo más cerca el procesamiento a los orígenes de datos, como más "al borde". Encontré otro par de referencias usando ésta palabra:

- [Intel: Computación de borde](https://www.intel.la/content/www/xl/es/edge-computing/what-is-edge-computing.html)
- [IBM: Computación de borde](https://www.ibm.com/mx-es/cloud/what-is-edge-computing)

Reply:
Podemos colocar la votación en [éste mensaje de Slack](https://cloud-native.slack.com/archives/C02PNU9BJBB/p1682439524249889).

Reply:
@marianogg9 en la reunión de hoy y con base en la [votación de Slack](https://cloud-native.slack.com/archives/C02PNU9BJBB/p1682439524249889) se decidió usar **Computación en el borde**.